_data/beyond-sidebar.yml

```diff
@@ -508,6 +519,12 @@
             - title: Create invoice
               link: https://api-docs.beyondshop.cloud/ng-order-order-public-api.html#resources-order-create-invoice
               id: create_invoice
+            - title: Create multiple invoices
+              link: https://api-docs.beyondshop.cloud/ng-order-order-public-api.html#resources-order-batch-actions-create-invoices
+              id: create_multiple_invoices
+            - title: Merge multiple invoices
```

Comment:
As discussed: I think "merge" is not the ideal verb here, as for me, "merge" also implies actions of overwriting or deleting existing content when 2 or more files are merged, something like "A+B=C", which is not the case here. I suggest "combine", which sounds more like "A+B = AB" to me (multiple files are combined in one document).